doc/source/ray-overview/examples.rst

```diff
@@ -311,27 +304,6 @@ Ray Examples
 
         Serving an Object Detection Model
 
-    .. grid-item-card:: :bdg-secondary:`Code example`
-        :class-item: gallery-item llm
-        :link: /ray-air/examples/dreambooth_finetuning
-        :link-type: doc
-
-        Fine-tuning DreamBooth with Ray AIR
-
-    .. grid-item-card:: :bdg-secondary:`Code example`
-        :class-item: gallery-item llm
-        :link: /ray-air/examples/stablediffusion_batch_prediction
-        :link-type: doc
-
-        Stable Diffusion Batch Prediction with Ray AIR
-
-    .. grid-item-card:: :bdg-secondary:`Code example`
```

Comment:
What about this?

Reply:
This is another duplicate: https://github.com/peytondmurray/ray/blob/918de4f5a15d3bfd7aaaf7c7ff4d83d067a1806f/doc/source/ray-overview/examples.rst?plain=1#L146